### ISSUE TITLE
Compatibility with Guardian 1.0.0

### ISSUE THREAD

**Issue by cdesch** [ueberauth/guardian_db#57]
Does anyone know if `guardian 1.0.0-beta` has any breaking changes that would require updating `guardian_db` to be compatible?

I was going to fork the project and see if I could get the current release to work with the latest guardian. I didn't know if there were any off the bat pitfalls.

```

Failed to use "guardian" (version 1.0.0-beta.0) because
  guardian_db (version 0.8.0) requires ~> 0.14
  mix.exs specifies ~> 1.0.0-beta

```

**Comment by hassox** [comment]
There is a small change that makes it incompatible. There is a PR that addresses this that you can use in the short term while we shepherd it through the release process.

The branch is: `update-docs-for-guardian-1.0`

**Comment by cdesch** [comment]
I see. trying to pull it now -- running into an error, but it might be just me:

config: 

```
{:guardian, "~> 1.0.0-beta.0"},
{:guardian_db, github: "ueberauth/guardian_db", branch: "update-docs-for-guardian-1.0"},

```
error:
```

cjs-MBP15:my_rewards_web cj$ mix deps.get
* Updating guardian_db (https://github.com/ueberauth/guardian_db.git)
** (Mix) Required version ">= 1.0" for package guardian is incorrectly specified (from: /Users/cj/elixir_projects/my_rewards_platform/deps/guardian_db/mix.exs)
```

**Comment by hassox** [comment]
Eep. Thanks for finding that issue. I will add a fix for that into the branch. In the meantime you can set override: true on the guardian dep to get you started. I will address this as quickly as I can.

Thank you for raising!

**Comment by cdesch** [comment]
It looks like `guardian_db` is looking for `{:guardian, ">= 1.0"},` instead of `{:guardian, "~> 1.0.0-beta.0"},`

I'm making a pull request to the branch

**Comment by doomspork** [comment]
Working on getting 1.0 released today w/ support for Guardian 1.0 @cdesch 👍

**Comment by cdesch** [comment]
thanks @doomspork
